rules/library.bzl

```diff
@@ -841,7 +841,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
     if swift_version:
         additional_swift_copts += ["-swift-version", swift_version]
 
-    module_data = library_tools["wrap_resources_in_filegroup"](name = module_name + "_data", srcs = data)
+    module_data = library_tools["wrap_resources_in_filegroup"](name = module_name + "_data", srcs = data, testonly = kwargs.get("testonly", False),)
```

Comment:
We should probably eventually refactor the `testonly = kwargs.get("testonly", False)` value out to some local scope since it's used so much (and should probably be used in even more places).